server/migrations/20220311011445_remove_cart_and_update_order.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema
+    .dropTable("cart_items")
+    .dropTable("cart")
+    .alterTable("orders", (table) => {
+      table.dropColumn("total_price");
+    })
+    .alterTable("orders", (table) => {
+      table.float("total_price", 0.0);
```

Comment:
this needs to be added to down (changing it back to interger maybe)

Reply:
yeah done that will push it with cart working